### PR TITLE
Importing Past Blog Posts

### PR DESCRIPTION
**Pull request by nerddtvg**

### CODE REVIEW

**Comment by nerddtvg** [comment]
Ready to merge.

File: src/pages/insights/randomize-slack.mdx [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+
+It would be great to expand this function to accept multiple users in the same workspace with registration via the Slack UI instead of the API website. That may be for a future blog post.
+
+## Reminder to Cleanup Resources
+
+As always, these are resources that cost money in Azure. Be sure to clean up the resources to prevent further billing once you're done using them. You should also delete the Slack App, or at a minimum revoke the `User OAuth Token` for security purposes.
+
+## Part Two
+
+Please check out [part two of this series](/insights/2021-07-22-randomize-slack-2): expanding to multiple users and multiple tenants!
```
Comment (Techneaux) [inline src/pages/insights/randomize-slack.mdx:152]: Link does not work

File: src/pages/insights/aws-config-compliance-automation.mdx [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+One way to help assist with all of these abilities, is to create an AWS Step Function surrounding the process. AWS Step Functions can document and help coordinate the workflows of tasks in many areas of AWS, but a very typical workflow can be orchestrated by pulling together AWS Lambda Functions into a coordinate set of tasks. 
+
+Storing metadata around the process is helpful too, and any persistence store desired can be used. AWS DynamoDB lends itself very well to simple data storage needs, and a sample set of information stored in DynamoDB which could assist a process such as compliance automation might look something like:
+
+| Field      | Description |
```
Comment (Techneaux) [inline src/pages/insights/aws-config-compliance-automation.mdx:69]: Might be nice if there was more spacing after tables. 
image

**Review by Techneaux — CHANGES_REQUESTED** [review]

File: src/pages/insights/aws-config-compliance-automation.mdx (hunk 2 above)
Comment (nerddtvg) [reply src/pages/insights/aws-config-compliance-automation.mdx:69]: To be fair, this blog post has been a bear. I blame @larryboymi

**Pull request merged**